.github/workflows/publish_github.yml

```diff
@@ -1,15 +1,12 @@
-name: Node.js Github Package
+name: Publish
```

Comment:
This is specifically publishing to GitHub registry, let's make it clear in the name.